pkg/wkspace/validator.go

```diff
@@ -32,3 +39,31 @@ func Preflight() (bool, error) {
 
 	return true, nil
 }
+
+func checkGitSSH() error {
+	dir, err := os.MkdirTemp("", "scaffolds")
+	if err != nil {
+		return err
+	}
+	defer func(path string) {
+		err := os.RemoveAll(path)
+		if err != nil {
+			err := utils.HighlightError(err)
+			if err != nil {
+				return
+			}
+		}
+	}(dir)
+
+	cmd := exec.Command("git", "clone", "git@github.com:pluralsh/scaffolds.git", dir)
+	// Configure the output to display progress as dots
+	output := &executor.OutputWriter{Delegate: os.Stdout}
```

Comment:
will this dump the full output in the case of error?  Will be helpful to explain to users.

Reply:
I have to check it

Reply:
now it dumps the error output